libnavui-dropin/src/main/res/layout/mapbox_info_panel_header_layout.xml

```diff
@@ -18,13 +18,20 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent" />
 
+        <androidx.constraintlayout.widget.Barrier
+            android:id="@+id/tripProgressGuideline"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:barrierDirection="start"
+            app:constraint_referenced_ids="startNavigation,endNavigation" />
+
         <FrameLayout
             android:id="@+id/tripProgressLayout"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
```

Comment:
🤔  if should set the width to `0dp`. That would stretch the frame layout to take the entire width starting from left edge of info panel upto the barrier. That way the customers when injecting there own trip progress view could decide the ggravity and align it to left/center/right. 

Whereas, with what we have now, it will always wrap content and be at the center. That would be fine for our design, but would not work if someone would want to have the trip progress left aligned.

@tomaszrybakiewicz

Reply:
Done